manifests/pki/ca.pp

```diff
@@ -17,18 +17,20 @@
 # }
 
 class strongswan::pki::ca (
-  $certificate_dir = '/etc/strongswan/ipsec.d/cacerts',
-  $private_key_dir = '/etc/strongswan/ipsec.d/private',
-  $common_name     = 'strongswanCA',
-  $country_code    = 'GB',
-  $organization    = 'Strongswan',
+  $ca_certificate_dir = $::strongswan::ca_certificate_dir,
+  $private_key_dir    = $::strongswan::private_key_dir,
+  $ipsec_dir          = $::strongswan::ipsec_d_dir,
+  $strongswan_dir     = $::strongswan::strongswan_dir,
```

Comment:
Please do not add `::` after `$`

Reply:
Sorry - not following?

Reply:
ah sorry, my mistake. Please reference variables like:
```puppet
$strongswan::strongswan_dir
```

Reply:
I kept with the existing style of the module so that is a bit unclear to me. Are you asking me to rename all of the variables in the module?

Reply:
On Vox Pupuli we had a longer discussion about this. Best practices and style guides change from time to time, how should this be handled with an existing codebase. We want to enforce the new rules for new code, even if this leads to a inconsistent codebase. In parallel to reviewing and merging PRs, we also go through modules and adapt the codestyle. We highly appreciate it if you are interested in adopting the style for the whole codebase, but that is not a requirement.